### ISSUE TITLE
Can't run app with just the library installed

### ISSUE THREAD

**Issue by Bernard996** [Assembless/react-native-material-you#6]
I wanted to try this library but if I install it, without even importing it in my project, I get this error:
`/home/bernard/MUI/node_modules/@assembless/react-native-material-you/android/src/main/java/com/assembless/reactnativematerialyou/RNMaterialYouModule.kt: (34, 24): Unresolved reference: system_accent1_0`
for all the color accents.

I'm using node v16.13.1 with react-native-cli: 2.0.1 react-native: 0.66.4

**Comment by DRFR0ST** [comment]
Are you trying to run the package on a non-Android 12 device? 

We're working on supporting fallback palette for devices that don't support Material You feature. See #5

**Comment by Bernard996** [comment]
No, I've tested it both on my Samsung Galaxy Z Fold 3 with Android 12 and an emulator

**Comment by DRFR0ST** [comment]
Unfortunately I wasn't been able to reproduce this problem on an emulator yet. I will try on a Samsung device hoping for luck. Thanks for reporting this!

**Comment by Bernard996** [comment]
No problem, if you want me to provide further information feel free to ask

Bernard
